src/debonair/Menu.java

```diff
@@ -0,0 +1,39 @@
+// user defined package
+package debonair;
```

Comment:
specified package doesn't show a relationship with other packages of the same program modify to something like 
```suggestion
package com.debonair.items;
```